modules/cucumber_testsuite/main.tf

```
@@ -30,12 +29,15 @@ locals {
   host_key => lookup(var.host_settings[host_key], "mac", null) if var.host_settings[host_key] != null }
   additional_repos = { for host_key in local.hosts :
   host_key => lookup(var.host_settings[host_key], "additional_repos", {}) if var.host_settings[host_key] != null }
+  images = { for host_key in local.hosts :
+  host_key => lookup(var.host_settings[host_key], "image", null) if var.host_settings[host_key] != null }
```

```suggestion
  host_key => lookup(var.host_settings[host_key], "image", "default") if var.host_settings[host_key] != null }
```
If the image value is not defined for the host, which default should we use? At least suse manager and suse manager proxy use "default" string as default value.

---

Others will use different defaults though. I'd leave it as it is for now